### PR TITLE
Update config.toml for release-1.19 branch

### PR DESCRIPTION
**Pull request by PI-Victor**
Signed-off-by: Victor Palade <victor@cloudflavor.io>

/assign @jimangel @irvifa @sftim 
/cc @reylejano 

~**p.s.: The `release-1.21` branch referenced here will be created on release day!**~ 
release-1.21 was created!

/hold till release day

### CODE REVIEW

**Comment by reylejano** [comment]
For reference, [latest patch releases](https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md)

**Comment by reylejano** [comment]
@PI-Victor , leave the configs for 1.18
https://kubernetes.io/docs/home/supported-doc-versions/ explains the documentation policy: documentation for the current version of Kubernetes and the four previous versions of Kubernetes.

File: config.toml [hunk 1]
```diff
@@ -183,41 +183,40 @@ js = [
...
 url = "https://v1-19.docs.kubernetes.io"
 
 [[params.versions]]
-fullversion = "v1.18.17"
+fullversion = "v1.18.20"
 version = "v1.18"
-githubbranch = "v1.18.17"
+githubbranch = "v1.18.20"
 docsbranch = "release-1.18"
 url = "https://v1-18.docs.kubernetes.io"
```
Comment (PI-Victor) [inline config.toml:219]: 1.18 is now added back

**Comment by annajung** [comment]
@reylejano @sftim will leave approval to you

**Comment by reylejano** [comment]
thank you for adding back 1.18
/approve

cancel the hold when ready

**Comment by PI-Victor** [comment]
need another approve here folks, seems prow was sleeping before

**Pull request merged**